src/build_workflow/build_recorder.py

```diff
@@ -60,6 +60,7 @@ def __init__(self, target: BuildTarget):
             self.data["build"]["version"] = target.opensearch_version
             self.data["build"]["platform"] = target.platform
             self.data["build"]["architecture"] = target.architecture
+            self.data["build"]["distribution"] = target.distribution if target.distribution else "None"
```

Comment:
Leave this as `target.distribution`, it should always have a value here coming from the caller.

Reply:
@dblock but if I leave as is, if no one call `--distribution` then it will default to `None`, which then will error out during recording.

Reply:
```
ValueError: Invalid manifest schema: {'build': [{'distribution': ['null value not allowed']}]}
```

Reply:
Set to `tar` if `None` for now.